dashboard/frontend/state/stores/runStore.js

```diff
@@ -0,0 +1,37 @@
+import { action, makeObservable, observable } from 'mobx';
+import Run from '../models/runModel';
+
+class RunStore {
+    @observable runs = [];
```

Comment:
If we're going to be accessing runs using ID, why not store it as a map instead?